packages/orbit-components/src/Accordion/index.js

```diff
@@ -0,0 +1,91 @@
+// @flow
+import * as React from "react";
+import styled from "styled-components";
+
+import Loading from "../Loading";
+import AccordionWrapper from "./components/AccordionWrapper";
+import defaultTheme from "../defaultTheme";
+import { Provider as SectionProvider } from "./AccordionContext";
+import getSpacingToken from "../common/getSpacingToken";
+
+import type { Props } from "./index";
+
+export const StyledAccordion = styled.div`
+  width: 100%;
+  box-sizing: border-box;
+  position: relative;
+  font-family: ${({ theme }) => theme.orbit.fontFamily};
+  margin-bottom: ${getSpacingToken};
+`;
+
+StyledAccordion.defaultProps = {
+  theme: defaultTheme,
+};
+
+const Accordion = ({ children, dataTest, spaceAfter, expanded, loading }: Props) => {
+  // Make first section expanded by default (index 0)
+  const [expandedSection, setExpandedSection] = React.useState(0);
+
+  React.useEffect(() => {
+    if (typeof expanded !== "undefined") {
+      setExpandedSection(expanded);
+    }
+  }, [expanded]);
+
+  const onDefaultExpand = React.useCallback(segmentId => setExpandedSection(segmentId), [
+    setExpandedSection,
+  ]);
+
+  const renderSection = item => {
+    if (React.isValidElement(item)) {
+      return React.cloneElement(item);
+    }
+
+    return null;
+  };
+
+  return (
+    <StyledAccordion spaceAfter={spaceAfter} data-test={dataTest}>
+      {children
+        ? React.Children.map(children, (item, key) => {
+            if (!item) return null;
+
+            const { id, onExpand } = item.props;
+
+            // This is used for the case when user wants to map sections and change their order
+            // related issue: #1005
+            const index = Number(item.key) || key;
+
+            // Either use provided id or item index
+            const sectionId = (typeof id !== "undefined" && id) || index;
+            // Determine if section is expanded
+            const isExpanded = expandedSection === sectionId;
```

Comment:
Probably here is a mistake:

You are comparing `expandedSection` which is `number` to sectionId which can be a `string`
1 === "1" will be false

Reply:
The idea is to have ability to pass string ids (e.g. flightId, segmentCode, ...). That's why we have `id` prop on AccordionSection component.

Reply:
We can use provided `AccordionSection` IDs to store in our state as expanded segment IDs. There shouldn't be any problem with that, right?